src/github-issues-prs.ts

```diff
@@ -104,10 +104,37 @@ export class GitHubIssuesPrsProvider implements TreeDataProvider<TreeItem> {
 			return false;
 		}
 
-		// take first one
-		await open(getGitHubUrl(remotes[0].url) + '/issues/new');
+		let urls: string[] = remotes.map(remote => remote.url);
 
-		return true;
+		window.showQuickPick(
+			Promise.resolve(urls),
+			{
+				placeHolder: 'Select the remote you want to create an issue on'
+			}
+		).then((remote: string|null) => {
+			if (!remote) {
+				return;
+			}
+
+
+			// get my object back
+			const selectedRemote = remotes.find(x => x.url === remote);
+			if(!selectedRemote) {
+				return;
+			}
+
+			const github = new GitHub();
+
+			github.repos.get({
```

Comment:
It does not support that, I'm getting the error that the method does not support an await statement, because it returns a Promise and is not an asynchronous function.